org.eclipse.jdt.core.javac/src/org/eclipse/jdt/core/dom/JavacConverter.java

```diff
@@ -261,12 +261,14 @@ private ExportsDirective convert(JCExports javac) {
 		commonSettings(res, javac);
 		List<JCExpression> mods = javac.getModuleNames();
 		if (mods != null) {
-			Iterator<JCExpression> it = mods.iterator();
-			while(it.hasNext()) {
-				JCExpression jcpe = it.next();
-				Expression e = convertExpression(jcpe);
-				if( e != null )
-					res.modules().add(e);
+			if (mods != null) {
```

Comment:
This check seems redundant; the previous line performs the same check

Reply:
My mistake, didn't see it. I will correct it.